### DISCUSSION TITLE
Why the form design is not the same as the ant-design react version？（表单设计与ant-design react版本不太一样这是为什么？）

### DISCUSSION THREAD

**Discussion by popring** [vueComponent/ant-design-vue#6066] [Q&A]
主要说的是这里的设计，在 ant-design 中 Form.Item 设置name后，表单控件无需设置 value 值，也无需设置 onChange ，数据同步被 form 接管。

<img width="1456" alt="image" src="https://user-images.githubusercontent.com/31247906/198907721-95327a79-6c90-4a0e-9e57-322333c81b27.png">

而ant-design-vue版本在设计上基本其他vue组件库基本一致，在 Form 和 FormItem 层面不会接管表单数据，只会用来做数据校验，具体数据同步还是与表单控件绑定。

代码使用层面区别

- ant-design

![image](https://user-images.githubusercontent.com/31247906/198907924-3fbd79ef-1692-4ff7-a9c5-e3c9c9394ff8.png)

- ant-design-vue

<img width="838" alt="image" src="https://user-images.githubusercontent.com/31247906/198908099-b973cc03-d499-42f5-88f4-d1972cd5a9f6.png">

**Comment by hminghe** [comment]
1.0版本是这么和react这么设计的， 基于element-ui是用户量比较大，大家都习惯了v-model的写法， 所以2.0开始也切换成v-model的写法了。
